Black-Market/Black-Market/Network/Services/UserNetworkService.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UserService.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 8/2/23.
+//
+
+import Foundation
+import Combine
+
+class UserNetworkService: NetworkService, UserNetworkInterface {
+    func signUp(params: [String : Any]) -> AnyPublisher<DefaultResponse, Error> {
+        getData(from: SignUpEndpoint(params: params))
+            .decode(type: DefaultResponse.self, decoder: JSONDecoder())
```

Comment:
given we are passing a new JSONDecoder most of the times, it probably make sense to make it a default param value.

Reply:
Done